ginga/gw/Desktop.py

```diff
@@ -212,13 +213,13 @@ def create_toplevel_ws(self, wsname, width, height,
         root = self.app.make_window()
 
         # TODO: this needs to be more sophisticated
-        ## root.set_title(wsname)
-        ws = self.make_ws(wsname, wstype='tabs', use_toolbar=True)
+        # root.set_title(wsname)
+        ws = self.make_ws(wsname, wstype='tabs', use_toolbar=True)  # noqa
 
         vbox = Widgets.VBox()
         vbox.set_border_width(0)
 
-        vbox.add_widget(bnch.widget)
+        #vbox.add_widget(bnch.widget)  # TODO: Undefined bnch!
```

Comment:
`bnch` is undefined in this method, and I can't figure out where it is supposed to come from, so I commented this.